src/examples/liquid_fill_gauge/liquid_fill_gauge.ts

```diff
@@ -250,7 +250,22 @@ const vis: LiquidFillGaugeVisualization = {
     this.svg.attr('height', element.clientHeight - 20)
 
     // @ts-ignore
-    d3.liquidfillgauge(this.svg, value, gaugeConfig)
+    if (details['print']) {
+      Object.assign(gaugeConfig, {
+        valueCountUp: false,
+        waveAnimateTime: 0,
+        waveRiseTime: 0,
+        waveAnimate: false,
+        waveRise: false
+      })
+      // @ts-ignore
+      d3.liquidfillgauge(this.svg, value, gaugeConfig)
```

Comment:
I don't want to be any obstruction, but since this is "example code" for people to follow, I think it's worth fretting over the details.

Since line 262 and 266 are identical, they should taken out of this "if" expression and run at the end (without an 'else' block)  The only bit of code that must be inside the `if() { block }` is the `Object.assign(guageConfig...`

And since a `return` statement at the end of a function is implied, it is not necessary to include it in the diff.

Reply:
Good call. Thanks for your input!